src/libraries/Microsoft.PowerFx.Core/Functions/IResourceManager.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    /// <summary>
+    /// Interface for resource manager.
+    /// </summary>
+    public interface IResourceManager
+    {
+        /// <summary>
+        /// Create a new resource element for string.
```

Comment:
>string [](http://example.com/codeflow?start=46&length=6)

What is this string? Can you elaborate on that? Is that the content, the URI, something else?

Reply:
Yes, the content. I'll update the comment